repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -351,3 +353,42 @@ export const completeMyFaresOtherProductsPages = (): void => {
         clickElementByText('Back');
     }
 };
+
+export const editServicesOtherProductsPage = (): void => {
+    for (let i = 0; i < 3; i++) {
+        clickRandomElementInTable('govuk-table__body', 'product-link');
+        getElementById('product-name').should('not.be.empty');
+        getElementById('product-status').should('not.be.empty');
+        getElementById('fare-type').should('not.be.empty');
+        clickElementById('Services-link');
```

Comment:
What happens if it clicks on a product which doesn't have services associated with it? e.g. period GeoZone tickets.

Reply:
fixed